src/errors/InvalidMappingError.ts

```diff
@@ -0,0 +1,10 @@
+import { Annotated } from './Annotated';
+
+export class InvalidMappingError extends Error implements Annotated {
+  specReferences = [
+    'https://www.hl7.org/fhir/elementdefinition-definitions.html#ElementDefinition.mapping'
+  ];
+  constructor() {
+    super('Invalid mapping, mapping.identity and mapping.map are 1..1 and must be set.');
```

Comment:
This error message focuses on the FHIR requirement being violated, but it might be more useful if it instead focused on what the FSH author would need to add in order to satisfy that requirement.

Reply:
Makes sense, I'll change that.